### PR TITLE
Add BBC Hindi NLI Dataset 

### PR DESCRIPTION
**Pull request by avinsit123**
# Dataset Card for BBC Hindi NLI Dataset

## Table of Contents
- [Dataset Description](#dataset-description)
  - [Dataset Summary](#dataset-summary)
  - [Supported Tasks](#supported-tasks-and-leaderboards)
  - [Languages](#languages)
- [Dataset Structure](#dataset-structure)
  - [Data Instances](#data-instances)
  - [Data Fields](#data-fields)
  - [Data Splits](#data-splits)
- [Dataset Creation](#dataset-creation)
  - [Curation Rationale](#curation-rationale)
  - [Source Data](#source-data)
  - [Annotations](#annotations)
  - [Personal and Sensitive Information](#personal-and-sensitive-information)
- [Considerations for Using the Data](#considerations-for-using-the-data)
  - [Social Impact of Dataset](#social-impact-of-dataset)
  - [Discussion of Biases](#discussion-of-biases)
  - [Other Known Limitations](#other-known-limitations)
- [Additional Information](#additional-information)
  - [Dataset Curators](#dataset-curators)
  - [Licensing Information](#licensing-information)
  - [Citation Information](#citation-information)

## Dataset Description

- HomePage : https://github.com/midas-research/hindi-nli-data
- Paper : "https://www.aclweb.org/anthology/2020.aacl-main.71"
- Point of Contact : https://github.com/midas-research/hindi-nli-data

### Dataset Summary

- Dataset for Natural Language Inference in Hindi Language. BBC Hindi Dataset consists of textual-entailment pairs.
- Each row of the Datasets if made up of 4 columns - Premise, Hypothesis, Label and Topic.
- Context and Hypothesis is written in Hindi while Entailment_Label is in English.
- Entailment_label is of 2 types - entailed and not-entailed.
- Dataset can be used to train models for Natural Language Inference tasks in Hindi Language.
[More Information Needed]

### Supported Tasks and Leaderboards

- Natural Language Inference for Hindi

### Languages

Dataset is in Hindi

## Dataset Structure

- Data is structured in TSV format. 
- Train and Test files are in seperate files

### Dataset Instances

An example of 'train' looks as follows.

```
{'hypothesis': 'यह खबर की सूचना है|', 'label': 'entailed', 'premise': 'गोपनीयता की नीति', 'topic': '1'}

```
### Data Fields

- Each row contatins 4 columns - Premise, Hypothesis, Label and Topic.

### Data Splits

- Train : 15553
- Valid : 2581
- Test : 2593

## Dataset Creation

- We employ a recasting technique from Poliak et al. (2018a,b) to convert publicly available BBC Hindi news text classification datasets in Hindi and pose them as TE problems
- In this recasting process, we build template hypotheses for each class in the label taxonomy
- Then, we pair the original annotated sentence with each of the template hypotheses to create TE samples.
- For more information on the recasting process, refer to paper "https://www.aclweb.org/anthology/2020.aacl-main.71"

### Source Data

Source Dataset for the recasting process is the BBC Hindi Headlines Dataset(https://github.com/NirantK/hindi2vec/releases/tag/bbc-hindi-v0.1)

#### Initial Data Collection and Normalization

-  BBC Hindi News Classification Dataset contains 4, 335 Hindi news headlines tagged across 14 categories: India, Pakistan,news, International, entertainment, sport, science, China, learning english, social, southasia, business, institutional, multimedia
-  We processed this dataset to combine two sets of relevant but low prevalence classes.
- Namely, we merged the samples from Pakistan, China, international, and southasia as one class called international.
- Likewise, we also merged samples from news, business, social, learning english, and institutional as news.
- Lastly, we also removed the class multimedia because there were very few samples.

#### Who are the source language producers?

Pls refer to this paper: "https://www.aclweb.org/anthology/2020.aacl-main.71"

#### Annotation process

Annotation process has been described in Dataset Creation Section.

#### Who are the annotators?

Annotation is done automatically.

### Personal and Sensitive Information

No Personal and Sensitive Information is mentioned in the Datasets.

## Considerations for Using the Data

Pls refer to this paper: https://www.aclweb.org/anthology/2020.aacl-main.71

### Discussion of Biases

Pls refer to this paper: https://www.aclweb.org/anthology/2020.aacl-main.71

### Other Known Limitations

No other known limitations

## Additional Information

Pls refer to this link: https://github.com/midas-research/hindi-nli-data

### Dataset Curators

It is written in the repo : https://github.com/avinsit123/hindi-nli-data that 
- This corpus can be used freely for research purposes.
- The paper listed below provide details of the creation and use of the corpus. If you use the corpus, then please cite the paper.
- If interested in commercial use of the corpus, send email to midas@iiitd.ac.in.
- If you use the corpus in a product or application, then please credit the authors and Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi appropriately. Also, if you send us an email, we will be thrilled to know about how you have used the corpus.
- Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi, India disclaims any responsibility for the use of the corpus and does not provide technical support. However, the contact listed above will be happy to respond to queries and clarifications.
- Rather than redistributing the corpus, please direct interested parties to this page
- Please feel free to send us an email:
  - with feedback regarding the corpus.
  - with information on how you have used the corpus.
  - if interested in having us analyze your data for natural language inference.
  - if interested in a collaborative research project.

### Licensing Information

Copyright (C) 2019 Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi (MIDAS, IIIT-Delhi).
Pls contact authors for any information on the dataset.

### Citation Information

```
    @inproceedings{uppal-etal-2020-two,
    title = "Two-Step Classification using Recasted Data for Low Resource Settings",
    author = "Uppal, Shagun  and
      Gupta, Vivek  and
      Swaminathan, Avinash  and
      Zhang, Haimin  and
      Mahata, Debanjan  and
      Gosangi, Rakesh  and
      Shah, Rajiv Ratn  and
      Stent, Amanda",
    booktitle = "Proceedings of the 1st Conference of the Asia-Pacific Chapter of the Association for Computational Linguistics and the 10th International Joint Conference on Natural Language Processing",
    month = dec,
    year = "2020",
    address = "Suzhou, China",
    publisher = "Association for Computational Linguistics",
    url = "https://www.aclweb.org/anthology/2020.aacl-main.71",
    pages = "706--719",
    abstract = "An NLP model{'}s ability to reason should be independent of language. Previous works utilize Natural Language Inference (NLI) to understand the reasoning ability of models, mostly focusing on high resource languages like English. To address scarcity of data in low-resource languages such as Hindi, we use data recasting to create NLI datasets for four existing text classification datasets. Through experiments, we show that our recasted dataset is devoid of statistical irregularities and spurious patterns. We further study the consistency in predictions of the textual entailment models and propose a consistency regulariser to remove pairwise-inconsistencies in predictions. We propose a novel two-step classification method which uses textual-entailment predictions for classification task. We further improve the performance by using a joint-objective for classification and textual entailment. We therefore highlight the benefits of data recasting and improvements on classification performance using our approach with supporting experimental results.",
}
```

### CODE REVIEW

File: datasets/BBC_Hindi_NLI/BBC_Hindi_NLI.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# coding=utf-8
```
Comment (lhoestq) [inline datasets/BBC_Hindi_NLI/BBC_Hindi_NLI.py:1]: could you move the files from the folder `BBC_Hindi_NLI` to `bbc_hindi_nli` ? We're using the snake case convention.
Could you also rename the dataset script accordingly ?

File: datasets/BBC_Hindi_NLI/BBC_Hindi_NLI.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "Context": datasets.Value("string"),
+                    "Hypothesis": datasets.Value("string"),
+                    "Entailment_Label": datasets.Value("string"),
+                    "Topic_Label": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/BBC_Hindi_NLI/BBC_Hindi_NLI.py:118]: Could you rename the fields `premise`, `hypothesis`, `label`, and `topic` ? This is the standard we use for NLI datasets.
Moreover can you use the ClassLabel feature type for label and topic ?
For example
```python
"label": datasets.ClassLabel(names=["neutral", "entailment", "contradiction"])
"topic": datasets.ClassLabel(names=["india", "pakistan", "news", ...])
```

your'll need to regenerate the dataset_infos.json file after these changes

**Review by lhoestq — COMMENTED** [review]
Nice ! Looking pretty good so far :)
Thanks for adding this one

I left a few comments

**Comment by lhoestq** [comment]
Hi @avinsit123 !
Did you manage to rename the dataset and apply the suggestion I mentioned for the data fields ?
Feel free to ping me when you're ready for a review :)

**Comment by lhoestq** [comment]
Hi @avinsit123 ! Have you had a chance to take a look at my suggestions ?
Let me know if you have questions or if I can help

**Comment by avinsit123** [comment]
@lhoestq sorry I completely forgot about this pr. I will complete it ASAP.

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.Value("string"),
+                    "topic": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/bbc_hindi_nli.py:124]:
```suggestion
                    "label": datasets.ClassLabel(names=["neutral", "entailment", "contradiction"]),
                    "topic": datasets.ClassLabel(names=["india", "pakistan", "news", ...]),
```
Could you use the ClassLabel feature type here please ?

You'll need to regenerate the dataset_infos.json after this change

File: datasets/bbc_hindi_nli/README.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+### Languages
+
+Dataset is in Hindi
+
+## Dataset Structure
+
+- Data is structured in TSV format. 
+- Train and Test files are in seperate files
+
+
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:75]: It looks like the `Data Instances` section is missing here, could you add it back ?
In this section you can add an example from the dataset.
To get one feel free to use `load_dataset` and take the first sample `dataset["train"][0]` for example

File: datasets/bbc_hindi_nli/README.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+### Data Fields
+
+- Each row contatins 4 columns - Context, Hypothesis, Entailment_Label and Topic_Label.
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:78]: Can you update the names of the columns here ?

File: datasets/bbc_hindi_nli/README.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:32]:
```suggestion
- [Dataset Structure](#dataset-structure)
  - [Data Instances](#data-instances)
  - [Data Fields](#data-fields)
  - [Data Splits](#data-splits)
```

File: datasets/bbc_hindi_nli/README.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+- The paper listed below provide details of the creation and use of the corpus. If you use the corpus, then please cite the paper.
+- If interested in commercial use of the corpus, send email to midas@iiitd.ac.in.
+- If you use the corpus in a product or application, then please credit the authors and Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi appropriately. Also, if you send us an email, we will be thrilled to know about how you have used the corpus.
+- Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi, India disclaims any responsibility for the use of the corpus and does not provide technical support. However, the contact listed above will be happy to respond to queries and clarifications.
+- Rather than redistributing the corpus, please direct interested parties to this page
+- Please feel free to send us an email:
+  - with feedback regarding the corpus.
+  - with information on how you have used the corpus.
+  - if interested in having us analyze your data for natural language inference.
+  - if interested in a collaborative research project.
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:151]: Is this a quote from the paper ? If so can you make it explicit by adding something like `It is written in the paper (link to the paper) that:` before the paragraph

File: datasets/bbc_hindi_nli/README.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+languages:
+- hi
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- extended|bbc__hindi_nli
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:15]: Here we expect to list the datasets that were used to construct this one.
As far as I understand this dataset contains original data from news:
```suggestion
- original
```

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for BBC Hindi NLI Config.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(bbc_hindi_nli_config, self).__init__(**kwargs)
+
+
+class bbc_hindi_nli(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/bbc_hindi_nli.py:104]:
```suggestion
class BbcHindiNli(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `bbc_hindi_nli`

**Review by lhoestq — COMMENTED** [review]
Thanks for changing the name !

I added a few suggestions:

File: datasets/bbc_hindi_nli/README.md (hunk 8 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md:15]: Not really so we have created the  NLI dataset from [BBC Hindi News Classification ](https://www.kaggle.com/c/learn-ai-bbc) So thats why I have used the extended tag. I have modified the dataset name accordingly.

File: datasets/bbc_hindi_nli/README.md (hunk 6 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md:32]: Done.

File: datasets/bbc_hindi_nli/README.md (hunk 5 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md:78]: Done.

File: datasets/bbc_hindi_nli/README.md (hunk 7 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md:151]: It was actually taken from a repo. I have added the repo link above the para.

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py (hunk 9 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/bbc_hindi_nli.py:104]: Done. Do I need to do this for `bbc_hindi_nli_config` also?

Comment (lhoestq) [reply datasets/bbc_hindi_nli/bbc_hindi_nli.py:104]: Yes it's better if they use the same convention

File: datasets/bbc_hindi_nli/README.md (hunk 4 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md:100]: Done.

Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md:100]: Added Data Instances

File: datasets/bbc_hindi_nli/dummy/bbc hindi nli/1.1.0/dummy_data/BBC_recasted_dev.tsv [hunk 10]
```diff
@@ -0,0 +1,5 @@
+premise	hypothesis	label	topic
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/dummy/bbc hindi nli/1.1.0/dummy_data/BBC_recasted_dev.tsv:1]: Hi ! Can you remove those tsv files please ?

File: datasets/bbc_hindi_nli/README.md (hunk 4 above)
Comment (lhoestq) [reply datasets/bbc_hindi_nli/README.md:100]: Can you move the `Data Instance` subsection inside the `Data Structure` section ?
It must follow the same order as in the table of contents

File: datasets/bbc_hindi_nli/README.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+<td>&nbsp;2</td>
+</tr>
+<tr>
+<td>&nbsp;बर्मा में मुसलमानों के खिलाफ हिंसा के और सुबूत</td>
+<td>&nbsp;यह भारत की सूचना नहीं है|</td>
+<td>&nbsp;entailed</td>
+<td>&nbsp;2</td>
+</tr>
+</tbody>
+</table>
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:93]: Moreover we expect an actual example as it is when you use the dataset in python.
You can use this for example:

````suggestion
```
{'hypothesis': 'यह खबर की सूचना है|',
 'label': 'entailed',
 'premise': 'गोपनीयता की नीति',
 'topic': '1'}
```
````

It's the first example of the train set

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py (hunk 3 above)
Comment (lhoestq) [reply datasets/bbc_hindi_nli/bbc_hindi_nli.py:124]: I just noticed that there is only entailment and not-entailment.
So I would suggest using 
```
"label": datasets.ClassLabel(names=["not-entailment", "entailment"]),
```
let me know what you think

**Review by lhoestq — COMMENTED** [review]
Thanks for the changes !
I added a few more comments:

File: datasets/bbc_hindi_nli/dummy/bbc hindi nli/1.1.0/dummy_data/BBC_recasted_dev.tsv (hunk 10 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/dummy/bbc hindi nli/1.1.0/dummy_data/BBC_recasted_dev.tsv]: Done

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.Value("string"),
+                    "topic": datasets.ClassLabel(names=["india", "news" , "international" , "sport" , "science"]),
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/bbc_hindi_nli.py:124]: It looks like one class is missing here no ?
In the csv file I can see labels from 0 to 5, but here there are only 5 labels (0 to 4).
There should be 6 labels.

File: datasets/bbc_hindi_nli/README.md [hunk 13]
```diff
@@ -0,0 +1,217 @@
...
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:93]:
```suggestion
```
you can remove that

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/bbc_hindi_nli.py:123]:
```suggestion
                    "label": datasets.ClassLabel(names=["not-entailment", "entailment"]),
```
you can use ClassLabel here as well

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+
+        with open(filepath, encoding="utf-8") as tsv_file:
+            tsv_reader = csv.reader(tsv_file, delimiter="\t")
+            for id_, row in enumerate(tsv_reader):
+                (premise, hypothesis, label, topic) = row
+                yield id_, {
+                    "premise": premise,
+                    "hypothesis": hypothesis,
+                    "label": label,
+                    "topic": topic,
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/bbc_hindi_nli.py:156]:
```suggestion
                    "label": 1 if label == "entailed" else 0,
                    "topic": int(topic),
```
For class labels we expect to have integers here

**Review by lhoestq — COMMENTED** [review]
Cools thanks :)
Apparently one label for the topic field is missing, can you add it ?
I added a few comments.
After these ones I think we should be good !

File: datasets/bbc_hindi_nli/README.md (hunk 4 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md:75]: Done

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py (hunk 15 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/bbc_hindi_nli.py]: Done. pls check

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py (hunk 14 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/bbc_hindi_nli.py]: Done. Pls check

File: datasets/bbc_hindi_nli/README.md (hunk 13 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/README.md]: Removed

File: datasets/bbc_hindi_nli/bbc_hindi_nli.py (hunk 12 above)
Comment (avinsit123) [reply datasets/bbc_hindi_nli/bbc_hindi_nli.py]: Added extra class. pls check

**Comment by avinsit123** [comment]
@lhoestq I have fixed the code to resolve all your comments. Pls do check. I also don't seem to know why the CI tests are failing as I ran all the tests in CONTRIBUTING.md on my local pc and they passed.

**Review by lhoestq — COMMENTED** [review]
Awesome thanks !
This looks all good to me :)

To fix the CI, I merged the master branch into your branch

**Comment by avinsit123** [comment]
@lhoestq thanks for ur patient review :) . I also wish to add similar 3 more NLI hindi datasets. Hope to do within this week.

File: datasets/bbc_hindi_nli/README.md [hunk 16]
```diff
@@ -0,0 +1,190 @@
...
+    url = "https://www.aclweb.org/anthology/2020.aacl-main.71",
+    pages = "706--719",
+    abstract = "An NLP model{'}s ability to reason should be independent of language. Previous works utilize Natural Language Inference (NLI) to understand the reasoning ability of models, mostly focusing on high resource languages like English. To address scarcity of data in low-resource languages such as Hindi, we use data recasting to create NLI datasets for four existing text classification datasets. Through experiments, we show that our recasted dataset is devoid of statistical irregularities and spurious patterns. We further study the consistency in predictions of the textual entailment models and propose a consistency regulariser to remove pairwise-inconsistencies in predictions. We propose a novel two-step classification method which uses textual-entailment predictions for classification task. We further improve the performance by using a joint-objective for classification and textual entailment. We therefore highlight the benefits of data recasting and improvements on classification performance using our approach with supporting experimental results.",
+}
+```
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:190]:
````suggestion
```

### Contributions

Thanks to [@avinsit123](https://github.com/avinsit123) for adding this dataset.
````
We added a new section :)

File: datasets/bbc_hindi_nli/README.md [hunk 17]
```diff
@@ -0,0 +1,190 @@
...
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:45]:
```suggestion
  - [Citation Information](#citation-information)
  - [Contributions](#contributions)
```

File: datasets/bbc_hindi_nli/README.md [hunk 18]
```diff
@@ -0,0 +1,190 @@
...
+
+## Dataset Description
+
+- HomePage : https://github.com/midas-research/hindi-nli-data
+- Paper : "https://www.aclweb.org/anthology/2020.aacl-main.71"
+- Point of Contact : https://github.com/midas-research/hindi-nli-data
```
Comment (lhoestq) [inline datasets/bbc_hindi_nli/README.md:51]:
```suggestion
- **Repository:** [GitHub](https://github.com/midas-research/hindi-nli-data)
- **Paper:** [Aclweb](https://www.aclweb.org/anthology/2020.aacl-main.71)
- **Point of Contact:** [GitHub](https://github.com/midas-research/hindi-nli-data)
```

**Review by lhoestq — APPROVED** [review]
Thanks for adding this one !

I just added a new section at the end of the README ;)

**Comment by avinsit123** [comment]
@lhoestq would this be merged to master?

**Comment by lhoestq** [comment]
Yes of course ;)
merging now !

**Pull request merged**